src/Network/GRPC/HighLevel/Generated.hs

```diff
@@ -28,10 +28,17 @@ module Network.GRPC.HighLevel.Generated (
 
   -- * Server Auth
 , ServerSSLConfig(..)
+
+  -- * Client
```

Comment:
I guess I won't touch it for now, then. 

Another thing we'll eventually need to communicate with our versioning is that we are tightly coupled to specific versions of the gRPC library. It's completely possible that we update our library to work with a new version of gRPC (and break compatibility with an old version) without changing the user-facing API at all. I guess that would correspond to a major version bump.

This reminds me that I need to show the gRPC library compatibility more prominently in the README. I will add it to this PR, since I'm writing docs anyway.